tests/Offer/CommandHandlers/ImportLabelsHandlerTest.php

```diff
@@ -16,19 +16,19 @@
 use CultuurNet\UDB3\Event\Events\LabelsImported;
 use CultuurNet\UDB3\Event\EventType;
 use CultuurNet\UDB3\Event\ValueObjects\LocationId;
-use CultuurNet\UDB3\Label;
+use CultuurNet\UDB3\Label as DeprecatedLabel;
```

Comment:
Nit: often the prefix `Legacy` is used. Not that important because this is inside the scope of one file but could be handy when searching for `LegacyLabel`. Same applies for `LegacyLabelName`

Reply:
Right, fixed!